packages/mui-base/src/PreviewCard/Positioner/usePreviewCardPositioner.types.ts

```diff
@@ -0,0 +1,131 @@
+import type * as React from 'react';
+import type {
+  Boundary,
+  Padding,
+  VirtualElement,
+  Side,
+  FloatingRootContext,
+  FloatingContext,
+} from '@floating-ui/react';
+import type { GenericHTMLProps } from '../../utils/types';
+
+export interface PreviewCardPositionerParameters {
+  /**
+   * If `true`, the preview card is open.
+   */
+  open?: boolean;
+  /**
+   * The anchor element to which the preview card popup will be placed at.
+   */
+  anchor?:
+    | Element
+    | null
+    | VirtualElement
+    | React.MutableRefObject<Element | null>
+    | (() => Element | VirtualElement | null);
+  /**
+   * The CSS position strategy for positioning the preview card popup element.
+   * @default 'absolute'
+   */
+  positionStrategy?: 'absolute' | 'fixed';
+  /**
+   * The container element to which the preview card popup will be appended to.
+   */
+  container?: HTMLElement | null | React.MutableRefObject<HTMLElement | null>;
+  /**
+   * The side of the anchor element that the preview card element should align to.
+   * @default 'bottom'
+   */
+  side?: Side;
```

Comment:
I think we should use rtl safe values here, like start/end. I know the type comes from Floating UI, but we can change the values on a component level.

Reply:
For example, for alignment, we use: start, center, and end. We would be consistent.

Reply:
You're right that we can support logical sides on a component level, but they are more complex to understand compared to physical sides, and logical alignments are far more useful for many cases, not only RTL. This is an issue for the anchor positioning hook though, so it shouldn't relate to this PR itself.